src/components/ebay-listbox/component.js

```diff
@@ -12,19 +12,17 @@ export default class {
     }
 
     handleChange(index, wasClicked) {
+        const details = { index, wasClicked };
         if (this.state.selectedIndex !== index) {
             const option = this.input.options[index];
-            const el = this.getEls('option')[index];
+            details.selected = [option.value];
+            details.el = this.getEls('option')[index];
             this.state.selectedIndex = index;
             this.once('update', () => {
-                this.emit('change', {
-                    index,
-                    selected: [option.value],
-                    el,
-                    wasClicked,
-                });
+                this.emit('change', details);
             });
         }
+        this.emit('select', details);
```

Comment:
This is needed, because `change` only fires when a *new* option is selected. If this is not included, the dropdown will not close after selecting the item that has already been picked.

Reply:
That should be fine. That is the existing behavior right?
We shouldn't need to fire any events if they select the same item.